RockWeb/Themes/NewSpring/Scripts/layout.js

```diff
@@ -30,8 +31,13 @@ document.addEventListener("DOMContentLoaded", function() {
 			content.style.marginBottom = navigationHeight + "px";
 
 			if(navigationSecondary) {
-				navigationSecondary.style.marginTop = 0;
-				content.style.marginTop = navigationSecondaryHeight + "px";
+				if(isAdmin) {
+					navigationSecondary.style.marginTop = 36 + "px";
```

Comment:
This feels a little funky. Can we not just do "36 px"?

Reply:
I mean, I guess? Does it matter one way or the other?

Reply:
It would just be syntactically similar if you used string literals like `"${navigationSecondaryHeight} px"` or something like that

Reply:
I don't know if I follow. that margin is being set statically because the height of the admin bar never changes. I suppose we could dynamically get the height of that element and use that instead in case it does change in the future?